crowbar_framework/app/models/api/storage.rb

```diff
@@ -15,6 +15,43 @@
 #
 
 module Api
-  class Storages
+  class Storage < Tableless
+    def repocheck
+      {}.tap do |ret|
+        [:ceph, :system].each do |r|
+          ret[r] = {
+            available: false
+          }
+        end
+
+        # check for the system repos
+        ["sles12-sp2-pool", "sles12-sp2-updates"].each do |repo|
```

Comment:
I'm a bit concerned that we're slowly hardcoding repos names everywhere, which will be a pain later on. Can't we somehow use some metadata from the repo yaml file instead to find the repos that need to be there?